cookbooks/ambari/attributes/default.rb

```diff
@@ -40,7 +40,8 @@
 
 
 node.default['ambari']['admin']['user'] = 'admin'
-node.default['ambari']['admin']['password'] = 'admin'
+node.default['ambari']['admin']['password'] = 'password'
+default['ambari']['admin']['default_password'] = 'admin'
```

Comment:
Is there a reason some of these are `node.default` and others are just `default` ?

Reply:
I have used `node.default` in ambari recipe but it is suggested that `default` is the style, used in chef-bach code. I have started using `default` and @amithkanand have replaced `node.default` with `default` as part of pull request #1182.